### DISCUSSION TITLE
Strange problem when loading the homepage

### DISCUSSION THREAD

**Discussion by AlexDernov** [neuefische/questions#730] [Web]
Problem with loading the homepage.
For the last few user stories I haven't changed anything on the homepage and suddenly it stopped loading and gives an error (Cannot read properties of null (reading 'lat')). At the same time "lat" is not present in my project!

[My PR.](https://github.com/AlexDernov/capestone-project-hungryOar/pull/21)

**Comment by NicoleHoerner** [comment]
Hi Alexandra!

There is not a lot of context of your error and I don't know what you have already investigated yet.

Do you have a map on your homepage using leaflet?

Maybe these two posts can help if it has do with leaflet:
https://github.com/geoman-io/leaflet-geoman/issues/249

https://github.com/Leaflet/Leaflet/issues/3280

This not really a solution from my side but hopefully it can help in anyway.

**Reply by AlexDernov** [reply]
Thank you very much!!!

**Comment by Kheirah** [comment]
There's a chance that it's the data that causes this issue, especially because you say you didn't change anything about the homepage. But you did add the functionality of adding new locations with coordinates. These coordinates would need to be correct for leaflet to show them on the map.

Have a look in the database if the coordinates are actually valid.

**Reply by AlexDernov** [reply]
👍🏻
